src/activity-functions-function-notation.xml

```diff
@@ -662,83 +578,221 @@
+            </solution>
           </webwork>
         </exercise>
 
-        <p>Earlier we mentioned the word <term>evaluate</term>. Again, this means to find the value of the output given an input value.</p>
+        <p>When looking at a graph it's easy to tell if the graph is that of a function or not.  Remember the inputs are found along the horizontal axis while the outputs are found along the vertical axis.  The rule is the same: one input, one unique output.</p>
+
+        <exercise>
+          <title>Determine if the graph is a function</title>
+          <webwork>
+            <setup>
+              <var name="$nopass">
+                <set>
+                  <member>\text{This graph is a function}</member>
+                  <member correct="yes">\text{This graph is not a function}</member>
+                </set>
+              </var>
+              <var name="$pass">
+                <set>
+                  <member correct="yes">\text{This graph is a function}</member>
+                  <member>\text{This graph is not a function}</member>
+                </set>
+              </var>
+              <pg-code>
+                $nopass = RadioButtons(["This graph is a function","This graph is not a function"],1);
+                $pass = RadioButtons(["This graph is a function","This graph is not a function"],0);
+              </pg-code>
+            </setup>
+            <statement>
+              <p>Drag the input left or right to see the results on the graph.</p>
+              <p>How many outputs does each input yeild?</p>
+              <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/QTfyBkEb/width/413/height/266/border/888888" width="413px" height="266px" style="border:0px;" width="431px" height="313px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
+              <p><var name="$nopass" form="buttons" /></p>
+              <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/Req8ZfCM/width/413/height/290/border/888888" width="413px" height="290px" style="border:0px;" width="431px" height="313px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
+              <p><var name="$pass" form="buttons" /></p>
+            </statement>
+          </webwork>
+        </exercise>
 
+        <p>The previous example leads us to a simple test we can use to determine if a graph is a fucntion or not.  It's called the <term>vertical line test</term><index><main>vertical line test</main></index>.</p>
+        <p>Given a graph, draw a vertical line through any place on the graph. If the vertical line crosses the graph at most one time (no matter where you draw the line), then the graph is a funciton. If the vertical line crosses the graph anywhere at more than one point, then the graph is not a function.</p>        
         <exercise>
```

Comment:
I don't understand what is happening with this exercise. It looks like it has no answer blanks.